routes/admin/portals/project-allotment-prof-create/index.js

```diff
@@ -157,23 +93,15 @@ function getProfessorType(profmail) {
                         admin: true
                     });
                 } else {
-                    projectHeadModel.findOne({
-                        instructor: profmail
+                    adminModel.findOne({
```

Comment:
Second query to the database is not required as the user can either be a `superUser` or a normal admin. If the user is not a superUser, simply return the department from the already fetched `result`. Thus, `result2` is not required.